fe1-web/src/core/components/QRCode.tsx

```diff
@@ -21,6 +21,7 @@ const styles = StyleSheet.create({
       and a height of 50%. centering this gives us 25% margin on every side
     */
     position: 'absolute',
+    padding: '4px',
```

Comment:
Remove the px (i.e. just write 4) and we‘re good to go 🚀😁 
  
https://reactnative.dev/docs/height-and-width

Reply:
Done